test/RdfParserTest.cpp

```diff
@@ -253,6 +256,10 @@ TEST(RdfParserTest, literalAndDatatypeToTripleComponent) {
   ASSERT_EQ(ladttc("+144321", fromIri(XSD_INTEGER_TYPE)), +144321);
   ASSERT_EQ(ladttc("true", fromIri(XSD_BOOLEAN_TYPE)), true);
   ASSERT_EQ(ladttc("false", fromIri(XSD_BOOLEAN_TYPE)), false);
+  auto result = ladttc("POINT(7.8 47.9)", fromIri(GEO_WKT_LITERAL));
+  auto vid = result.toValueIdIfNotString();
+  ASSERT_TRUE(vid.has_value() &&
+              vid.value().getDatatype() == Datatype::GeoPoint);
```

Comment:
I think for the test coverage you also need a WKT-Literal that is not a point to test that `else` branch.